django_drf_filepond/api.py

```diff
@@ -116,11 +116,32 @@ def store_upload(upload_id, destination_file_path):
     if storage_backend:
         return _store_upload_remote(destination_path, destination_name, tu)
     else:
-        return _store_upload_local(destination_path, destination_name, tu)
+        return _store_upload_local(destination_path, destination_name, tu,
+                                   rename_if_exists=rename_if_exists)
+
+
+def _get_available_target_file_path(target_dir, target_filename,
+                                    rename_if_exists=False):
+    target_file_path = os.path.join(target_dir, target_filename)
+    if not os.path.exists(target_file_path):
+        return target_file_path
+
+    if rename_if_exists:
+        idx = 2
+        while os.path.exists(target_file_path):
+            root, ext = os.path.splitext(target_filename)
+            target_file_path = f'{root} ({idx}).{ext}'
```

Comment:
I believe `splitext` includes the `'.'` in the _ext_ string. As a result I think this will result in a filename with two '.'s. I'd also prefer to avoid spaces in filenames if possible but I guess this may be just personal preference. Happy to update this after merging.